guides/common/modules/proc_disabling-cockpit-on-project.adoc

```diff
@@ -0,0 +1,30 @@
+[id="disabling-cockpit-on-project_{context}"]
+= Disabling {CockpitTitle} on {Project}
+
+Perform the following procedure if you want to disable {Cockpit} on {Project}.
+
+.Procedure
+. Disable {Cockpit} with the `{foreman-installer}` command:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{foreman-installer} --no-enable-foreman-plugin-remote-execution-cockpit
+----
+. In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *Remote execution*.
+. Erase the value of the *Cockpit URL* setting.
+. Uninstall the {Cockpit} package from {Project}:
```

Comment:
If we support uninstalling, I wonder if we should enhance the installer to do this. Technically it's possible to ensure a state of a package. Right now there's no parameter to ensure it's absent, but we can implement it.

Reply:
It was mentioned in the BZ that it is not probable that the fix will be coming anytime soon. In the meantime it would be good to have a workaround.

Reply:
I thought it should be easy to write, so I did: https://github.com/theforeman/puppet-foreman/pull/1111. I must admit I didn't test it yet, but it should work with `--foreman-plugin-remote-execution-cockpit-ensure absent`.

Reply:
Understood. I will hold off working on this PR then.

Reply:
I'm not sure if it will land in Foreman 3.6 and certainly not in 3.5, so it may be ok to document this for 3.5. Though I do think we can do better in supporting functionality in the tool itself rather than documenting "workarounds".

Reply:
I of course agree with that.

Reply:
So, what is the next step? Is this procedure good to go?

Reply:
I think so.